chaco/plot_containers.py

```diff
@@ -42,6 +42,7 @@
     simple_container_get_preferred_size,
     simple_container_do_layout,
 )
+from enable.stacked_layout import stack_layout, stacked_preferred_size
```

Comment:
we can use `enable.api` once https://github.com/enthought/enable/pull/840 is merged.